main.go

```diff
@@ -92,7 +94,13 @@ func mainCmd(args []string) int {
 	if err := checker.CheckPackages(paths...); err != nil {
 		if e, ok := err.(errcheck.UncheckedErrors); ok {
 			for _, uncheckedError := range e.Errors {
```

Comment:
This level of nesting is getting pretty horrible. How about moving this to `func reportUncheckedErrors(e errcheck.UncheckedErrors)` ?